venafi/resource_venafi_certificate_test.go

```diff
@@ -679,15 +753,15 @@ func TestTokenECDSASignedCert(t *testing.T) {
 				Config: config,
 				Check: func(s *terraform.State) error {
 					t.Log("Issuing TPP certificate with CN", data.cn)
-					return checkStandardCert(t, &data, s)
+					return checkStandardCertPKCS1(t, &data, s)
 				},
 			},
 			r.TestStep{
 				Config: config,
 				Check: func(s *terraform.State) error {
 					t.Log("Testing TPP certificate second run")
 					gotSerial := data.serial
-					err := checkStandardCert(t, &data, s)
+					err := checkStandardCertPKCS1(t, &data, s)
```

Comment:
Mostly to make verbose the function name before getting inspecting it. Actually if the concern is about preserving the standard name `checkStandardCert`, we could rename the `checkStandardCertPKCS8` instead since PKCS8 is our new standard. What do you think?